src/input/input_mp4.cpp

```diff
@@ -196,7 +196,12 @@ namespace Mist{
       std::cerr << "Input from stdin not yet supported" << std::endl;
       return false;
     }
-    if (config->getBool("headeronly")){return true;}
+    std::string dtshOutUrl = config->getString("dtshoutput");
+    if (!config->getBool("headeronly") && !dtshOutUrl.empty() > 0){
```

Comment:
Nitpick: `.empty()` returns a boolean, so for readability you might as well write `&& !dtshOutUrl.empty()` or `&& dtshOutUrl.empty() == false` (now I had to lookup what `.empty()` actually returns cause the `> 0` would imply it does not simply return a boolean)